gpu_bdb/benchmark_runner/slurm/run.sh

```diff
@@ -2,7 +2,8 @@
 
 ACCOUNT=rapids
 PARTITION=partition
-WORKERS=1
+NODES=1
+let "WORKER_NODES = $NODES - 1"
 
 IMAGE="/path/to/container/"
```

Comment:
In the interests of our team using this directly, it'd be fine to have a default working container path (when there is one) on Selene, and have a comment that using outside Selene will require changing the path.

Reply:
Resolved by https://github.com/rapidsai/gpu-bdb/pull/185/commits/45d675c755f8ac6206609c50b7980e087eaea695